litr/src/main/java/com/linkedin/android/litr/io/MediaMuxerMediaTarget.java

```diff
@@ -40,16 +45,51 @@
 
     private MediaFormat[] mediaFormatsToAdd;
 
+    private ParcelFileDescriptor parcelFileDescriptor;
     private String outputFilePath;
     private int numberOfTracksToAdd;
     private int trackCount;
 
+    /**
+     * This constructor to support scoped-storage enforcement in Android 10+, by using Uri instead of directly writing to File
```

Comment:
Documentation should change to:
Create an instance using input URI. On Android Oreo (API level 26) and above it can be any URI writeable by the provided context, otherwise it must be a "file://" URI. This constructor is especially useful when working with FileProvider created URI to comply with scoped storage enforcement on Android 10+